jdaviz/core/events.py

```diff
@@ -51,14 +51,14 @@ def path(self):
 
 
 class DataSelectedMessage(Message):
-    def __init__(self, indicies, *args, **kwargs):
+    def __init__(self, indices, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self._indicies = indicies
+        self._indices = indices
 
     @property
-    def indicies(self):
-        return self._indicies
+    def indices(self):
```

Comment:
This does actually change the API... @rosteen @javerbukh @ojustino , any objection on this change?

Reply:
I didn't see where this is used anywhere.  And I assume that external users are using `DataSelectedMessage`?

Reply:
Which begs the question -- is `DataSelectedMessage` even needed?

Reply:
This looks like an orphan message type that may have been used before, but can be removed now (as far as I can tell too).

Reply:
🤷

Reply:
Should I remove `DataSelectedMessage` in a separate PR or here?  It's a simple change, but not really a typo fix.

Reply:
Might as well just do it here? One less PR to review... 😬

Reply:
There appears to be several unused `*Message` classes, so I opened a new PR:  https://github.com/spacetelescope/jdaviz/pull/873